opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -350,6 +354,18 @@ def _is_valid_attribute_value(value: types.AttributeValue) -> bool:
     return True
 
 
+def _filter_attribute_values(attributes: types.Attributes):
+    if attributes:
+        for attr_key, attr_value in list(attributes.items()):
+            if _is_valid_attribute_value(attr_value):
+                if isinstance(attr_value, MutableSequence):
```

Comment:
Does the test cover the `MutableSequence` case?

Reply:
Should already be done [here](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-sdk/tests/trace/test_trace.py#L572)